passes/techmap/techmap.cc

```diff
@@ -1032,6 +1079,14 @@ struct TechmapPass : public Pass {
 		log("        former has a 1-bit for each constant input bit and the latter has the\n");
 		log("        value for this bit. The unused bits of the latter are set to undef (x).\n");
 		log("\n");
+		log("    _TECHMAP_WIREINIT_<port-name>_\n");
+		log("        When a parameter with this name exists, it will be set to the initial\n");
+		log("        value of the wire(s) connected to the given port, as specified by the init\n");
+		log("        attribute. If the attribute doesn't exist, x will be filled for the\n");
+		log("        missing bits. If this module is used for mapping, the init attribute bits\n");
```

Comment:
If this parameter is present, does this makes it mandatory for the rule to consume it (because otherwise it will get blown away?) This seems like quite an onerous things to do...

Reply:
Perhaps we could have a `_TECHMAP_REMOVEINIT_x_` wire, where a 1 in a bit of this wire removes `(* init *)` from that bit of port x. This is similar to how wires trigger actions in techmap more generally.

Reply:
@daveshah1 that sounds like a good idea, I'll implement it.